wagtailimporter/serializer.py

```diff
@@ -226,39 +236,33 @@ def db_filename(self):
         """
 
         folder_name = 'original_images'
-        filename = ('images/%s' % self.file).replace('/', '')
+        filename = 'images/%s' % self.file.replace('/', '-')
```

Comment:
Maybe `slugify` would be better?

Reply:
As this is trying to import an already existing file, `self.file` will either be a valid filename, or the developer is trying to import a file that can't possibly exist in the first place, at which point they have bigger issues to deal with!